prov/gni/include/gnix_nic.h

```diff
@@ -296,16 +326,18 @@ struct gnix_smsg_amo_cntr_hdr {
  */
 struct gnix_tx_descriptor {
 	struct dlist_entry          list;
+	struct slist_entry	    slist;
```

Comment:
> Also I have not traced the use of the list and slist fields throughout, but is it possible to just use list or are they in use simultaneously?

It looks like the `list` field is used for maintaining a free list of transmission descriptors.  So, they are being used simultaneously.  Also, I only needed a singly linked list for keeping track of the outstanding rdma/fma txd's.